cider/src/debugger/commands/path_parser.pest

```diff
@@ -1,13 +1,13 @@
 root = { "." }
 
-separator = { "-" }
+seperator = { "-" }
 
 body = { "b" }
 
 num = { ASCII_DIGIT+ }
 
-branch = {"t" | "f"}
+branch = { "t" | "f" }
 
-clause = { separator ~ (body | num | branch) }
+clause = { seperator ~ (body | num | branch) }
 
-path = { SOI ~ root ~ clause* ~ EOI }
+path = { SOI ~ num ~ root ~ clause* ~ EOI }
```

Comment:
Seems like the `name` field can only be a number right now. We want to make this
parse actual component names